rules/filename-case.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+var path = require('path');
```

Comment:
Sounds like a good proposal. But for now, the `import plugin` is not code-style oriented, though with `import-order`'s rule, it will start.